portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,11 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+async function loadingcontent() {
+    const responseFromServer = await fetch('/hello');
```

Comment:
nit maybe rename to serverResponse instead, since it's shorter but still clear what it is.

Reply:
Done